README.md

```diff
@@ -1,78 +1,45 @@
-# Thank You For Your Interest In CyberFortress
+# CyberFortress
 
-If you're reading this, our company has definitely piqued your interest. 
-We're glad you have made it this far and we hope to get to know you more!
-Please follow the instructions to submit your application.
+### Setup
 
-As per our [culture](https://www.cyberfortress.com/about/culture/), 
-we base all merit upon what you are capable of and not upon
-any certifications or education. Being counter-culture, consider this your
-resume. Given that, we thought that this would 
-be a creative way to express that and allow you, the applicant, 
-to express yourself. 
+* Node v12 is assumed, you can use [NVM](https://github.com/nvm-sh/nvm#install--update-script) to install if needed
+* Ensure that the [MongoDB](https://www.mongodb.com/download-center/community) community server is installed
+* Run `npm install` and `npm run db:seed` from the app directory
+* Create the env file with `cp .env.example .env`, I will provide my Geocodio and Darksky keys via email
 
-## Instructions
+### Using the app
 
-Fork this repository and follow the scenario below.
-The user data is located in `dataset.csv`.
-Be sure to meet all the objectives.
-When you are finished, open a PR against this repository and we will review
-it and reach out to you via PR comments.
-Take as long as you need. 
-We prefer quality work even if it takes extra time.
+Use `npm start` to run the app. You can then send requests using curl or Postman.
 
-## Scenario
+###### Register an admin user
+```
+curl -X POST 'http://localhost:8080/admin/signup?email=test@test.com&password=password'
+```
 
-Your mission, should you choose to accept it, is to build an API which, 
-given a user's unique identifier, should tell us whether it is raining
-in the location where the user lives.
+###### Login with your new admin user
+```
+curl -X POST 'http://localhost:8080/admin/login?email=test@test.com&password=password'
```

Comment:
Why is the request method a POST with no request data?

Reply:
The request data is in the query string, I suppose technically that should be in the body for a POST.